test/framework/main/stability.json

```diff
@@ -221,6 +221,90 @@
                     ]
                 }
             ]
+        },
+        {
+            "name": "cksums_ipv4_tcp_vlan",
+            "test-time": 60000000000,
+            "test-type": "TestTypeScenario",
+            "test-apps": [
+                {
+                    "image-name": "nff-go-test-cksum",
+                    "app-type": "TestAppGo",
+                    "exec-cmd": [
+                        "./testCksum", "-testScenario=1", "-hwol", "-ipv4", "-tcp", "-vlan",  "-number=1000000", "-inport=0", "-outport=1"
```

Comment:
Do we need also test configuration with no hwol flag in 1st part (only in 2nd)?

Reply:
We want checks HW vs SW Checksums in both part 1 and part 2.
I will add -hwol to 1st part in not-vlan tests